### PR TITLE
travis native deploy and depoly only when to master or feature

### LINKED ISSUE #2: investigate native travis deploy
https://docs.travis-ci.com/user/deployment/pages/

### LINKED ISSUE #7: limit deployment only to commits to master branch

### PR DESCRIPTION
**Pull request by marwin1991**
Fixes #2 
Fixes #7 

refering to #7 i decided to also add ^feature, is should depoly when go commit to master or branch starts with feature, other branches won't trigger build, only start npm run build to check if build not fails.
I added feature branches to enable testing in web when someone do not have configured env.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,9 +2,17 @@ language: node_js
...
+
+deploy:
+  provider: pages
+  skip-cleanup: true
+  github-token: $GH_TOKEN
+  keep-history: true
+  local-dir: ./dist
+  on:
+    all_branches: true
+    condition: $TRAVIS_BRANCH =~ master|^feature$
```
Comment (grzanka) [inline .travis.yml]: What does this condition mean ? Add a line with comments

File: .travis.yml [hunk 2]
```diff
@@ -2,9 +2,17 @@ language: node_js
...
-  - npm run deploy
+install:
+  - npm install
+script:
+  - npm run build
+
+deploy:
+  provider: pages
+  skip-cleanup: true
+  github-token: $GH_TOKEN
```
Comment (grzanka) [inline .travis.yml]: Where is GH_TOKEN being defined ?

Comment (marwin1991) [reply .travis.yml]: in travis/settings/Environment Variables

File: .travis.yml (hunk 1 above)
Comment (marwin1991) [reply .travis.yml]: depoly when commit to branch master or starting with feature
comment added

File: .travis.yml [hunk 3]
```diff
@@ -2,9 +2,17 @@ language: node_js
...
+
+deploy:
+  provider: pages
+  skip-cleanup: true
+  github-token: $GH_TOKEN # defined at Travis/Settings/Environment Variables
+  keep-history: true
+  local-dir: ./dist
+  on:
+    all_branches: true
+    condition: $TRAVIS_BRANCH =~ master|^feature #depoly when commit to branch master or starting with feature
```
Comment (grzanka) [inline .travis.yml]: depoly ?

Comment (marwin1991) [reply .travis.yml]: misspelled, fixed

**Pull request merged**